### DISCUSSION TITLE
HTML page with multiple download formats

### DISCUSSION THREAD

**Discussion by Achyuth072** [ImranR98/Obtainium#790] [Q&A] [answered]
So I was thinking of adding Stremio updates to Obtanium, but their downloads page has the downloads for all platforms.
I know with multiple download options it'll pick one alphabetically. But I don't know what its picking.
Heres the link : 
https://www.stremio.com/downloads

**Comment by ImranR98** [comment, marked as answer]
You can use the link filter and/or apk filter to choose a specific file. Probably adding `arm64` to the apk filter will work.
